src/components/Calculator.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
+export default class Calculator extends React.Component {
```

Comment:
- [ ] Please use ES6 syntax for this project. You can change this line of code to previous version and Please use arrow functions where applicable.